### PR TITLE
Simple "add dependent" form

### PR DESCRIPTION
**Pull request by forevermatt**
### Added
- Add a simple "add-dependent" form

### CODE REVIEW

File: src/components/DependentForm.svelte [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const dispatch = createEventDispatcher()
+
+const onCancel = () => {
+  dispatch('cancel')
+}
+const onSubmit = () => {
+  dispatch('submit', { name })
+}
+
+let name = ''
```
Comment (hobbitronics) [inline src/components/DependentForm.svelte:14]: Just a note on where we want things declared. Are you going with const first, whether functions or not?

File: src/components/DependentForm.svelte [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<div class="w-50">
+  <Form on:submit={onSubmit}>
+    <TextField label="Dependent Name" bind:value={name} autofocus />
+    <p>
+      Dependents include non-member spouses and children under 26 who haven't
+      married or finished college. Coverage for dependents is limited to $3,000
+      per person.
+    </p>
+    <Button on:click={onCancel}>Cancel</Button>
+    <Button raised>Save</Button>
```
Comment (hobbitronics) [inline src/components/DependentForm.svelte:26]: I added a capitalize class to ui-components, so we will probably want to do that here and add 
`.capitalize {
	text-transform: capitalize;
}` to this file for now.

Comment (hobbitronics) [reply src/components/DependentForm.svelte:26]: until we publish the ui-components repo that is.

File: src/pages/household/settings/dependent.svelte [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<script>
+import { goto } from '@roxi/routify'
+import { Page } from '@silintl/ui-components'
+import DependentForm from '../../../components/DependentForm.svelte'
```
Comment (captaincrazybro) [inline src/pages/household/settings/dependent.svelte:4]: Have a consistent, alphabetic order of organizing the imports. For WeCarry we put the local imports (ones where you have to navigate to the file in your projects) first and then the package imports.

File: src/pages/household/settings/dependent.svelte [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+const onCancel = () => {
+  $goto('../settings')
```
Comment (hobbitronics) [inline src/pages/household/settings/dependent.svelte:7]: is /settings what you want here? maybe this works, I assume you tested.

**Review by hobbitronics — APPROVED** [review]

File: src/pages/household/settings/dependent.svelte (hunk 4 above)
Comment (forevermatt) [reply src/pages/household/settings/dependent.svelte:7]: It does work, thanks. It seemed unintuitive, but this is apparently the syntax for going up that level.

File: src/components/DependentForm.svelte (hunk 1 above)
Comment (forevermatt) [reply src/components/DependentForm.svelte:14]: I was yes. Thoughts?

File: src/pages/household/settings/dependent.svelte (hunk 3 above)
Comment (forevermatt) [reply src/pages/household/settings/dependent.svelte:4]: Okay, great. I'll do that, thanks.

**Pull request merged**